app/controllers/episode_segmenter_controller.rb

```diff
@@ -1,6 +1,48 @@
 class EpisodeSegmenterController < ApplicationController
+  before_action :set_episode
+
   def show
+    authorize @episode, :show?
+    @uncut&.assign_attributes(uncut_params)
+  end
+
+  def update
+    authorize @episode, :update?
+    @uncut&.assign_attributes(uncut_params)
+
+    respond_to do |format|
+      if @uncut&.save
+        @uncut.slice_contents!
+        @episode.copy_media
```

Comment:
Sort of - `.slice_contents!` creates new `episode.contents` (the individual segments) ... but doesn't kick off any jobs to actually Porter-process them.  So we call `.copy_media` every time just in case we need to kick that off.

And if a job already exists, this is a no-op.